attachment_s3/models/ir_attachment.py

```diff
@@ -78,6 +66,25 @@ def _get_s3_bucket(self, name=None):
                     ) % (bucket_name, bucket_name)
 
             raise exceptions.UserError(msg)
```

Comment:
Shouldn't we raise this exception in _get_s3_bucket after we call `_get_s3_connection_params()`?
It allows a chance to call `super()` and adjust the parameters afterwards without getting an exception.

Reply:
hmm you make a good point. I agree.